libs/island-ui/core/src/lib/IconRC/iconMap.ts

```diff
@@ -36,6 +36,7 @@ export type Icon =
   | 'heart'
   | 'home'
   | 'informationCircle'
+  | 'links'
```

Comment:
This icon's name is `link`, not `links` 😅

<img width="578" alt="Screenshot 2022-06-29 at 08 50 27" src="https://user-images.githubusercontent.com/3789875/176394656-b6189dce-6c92-444f-acf5-3e3232686dd0.png">

Reply:
I took the name straight from figma. Where is your screen-shot taken from?
<img width="405" alt="Screenshot 2022-06-29 at 09 40 10" src="https://user-images.githubusercontent.com/5700902/176405281-4328f208-3d53-4d4b-8a23-63c8a024b49d.png">